torch/testing/_internal/common_methods_invocations.py

```diff
@@ -276,6 +276,8 @@ def method_tests():
         ('clamp', (), (None, 0.5), 'min_scalar', (True,)),
         ('clamp', (), (0.5, None), 'max_scalar', (True,)),
         ('clamp', (S, S), (), 'max_scalar_kwarg', (True,), (), (), ident, {'max': 1}),
+        ('clip', (S, S, S), dont_convert((0, 1)), '', (False,)),
+        ('clip_', (S, S, S), dont_convert((0, 1)), '', (False,)),
```

Comment:
before your additions of `absolute_` and `clip_` looks like inplace versions weren't tested here, so maybe they shouldn't be?

Reply:
The alias test requires an inplace entry.